python/hail/expr/aggregators.py

```diff
@@ -1087,3 +1088,82 @@ def info_score(gp) -> StructExpression:
     """
     t = hl.tstruct(score=hl.tfloat64, n_included=hl.tint32)
     return _agg_func('infoScore', gp, t)
+
+
+@typecheck(y=agg_expr(expr_float64),
+           x=oneof(expr_float64, sequenceof(expr_float64)))
+def linreg(y, x):
+    """Compute linear regression statistics.
+
+    Examples
+    --------
+    Regress HT against an intercept (1) , SEX, and C1:
+
+    >>> table1.aggregate(agg.linreg(table1.HT, [1, table1.SEX == 'F', table1.C1]))
+    Struct(beta=[88.50000000000014, 81.50000000000057, -10.000000000000068],
+    standard_error=[14.430869689661844, 59.70552738231206, 7.000000000000016],
+    t_stat=[6.132686518775844, 1.365032746099571, -1.428571428571435],
+    p_value=[0.10290201427537926, 0.40250974549499974, 0.3888002244284281],
+    n=4)
+
+    Regress blood pressure against an intercept (1), age, height, and height squared:
+
+    >>> ds_ann = ds.annotate_rows(linreg = hl.agg.linreg(ds.pheno.blood_pressure,
```

Comment:
move `linreg = ...` to new line so you can keep the x list on one row

nice example!